src/main/java/com/google/devtools/build/lib/BUILD

```diff
@@ -413,13 +413,26 @@ java_library(
     ],
 )
 
```

Comment:
This change seems fine -- but I'm not quite clear on why it's needed. Could we perhaps leave a comment here as to why this particular class is special and needs to be excluded from the syntax target?

Reply:
In the next change, I add a dependency on skylark-semantics to several targets in order to silence the warnings about "no enum constant" that happen when we build. At that point it should be fairly obvious, but I can add a comment explaining this if you'd prefer.

Reply:
OK -- your discretion. If it'll be fairly obvious, no need to bother.